dependencies.gradle

```diff
@@ -17,5 +17,7 @@ allprojects {
         MATERIAL_VERSION = "1.2.1"
         POWERMOCK_VERSION = "2.0.9"
         AGP_VERSION = "4.1.2"
+
+        KOTILN_VERSION = "1.3.72"
```

Comment:
Small typo in `KOTILN_VERSION` - should be `KOTLIN_VERSION`

Reply:
Done.